jest.config.js

```diff
@@ -1,5 +1,31 @@
+const esModules = [
+  "bail",
+  "comma-separated-tokens",
+  "decode-named-character-reference",
+  "devlop",
+  "estree-util-.+",
+  "hast-util-.+",
+  "html-url-attributes",
+  "is-plain-obj",
+  "lodash-es",
+  "mdast-util-.+",
+  "micromark",
+  "property-information",
+  "react-base16-styling",
+  "react-json-tree",
+  "react-markdown",
+  "remark-.+",
+  "space-separated-tokens",
+  "trim-lines",
+  "trough",
+  "unified",
+  "unist-.+",
+  "vfile",
+  "vfile-message",
+].join("|");
+
 export default {
-  preset: "ts-jest",
+  preset: "ts-jest/presets/js-with-babel",
```

Comment:
This is needed to transform `.js` files in `node_modules` to commonjs syntax in order for tests to run properly. The `ts-jest` preset on its own did not transform `.js` files, so our tests choked when importing from ESM libraries.